README.md

```diff
@@ -32,13 +32,23 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+    The DOM or Document Object Model is an API that represents an HTML document as a tree structure comprised of node objects.
+
 2. What is an event?
 
+    Events are actions that take place within a program environment.
+
 3. What is an event listener?
 
+    Event Listeners allow us to detect specific events that take place in a progrmimng evniroment and preform a given set of actions once the event we are listening occurs.
+
 4. Why would we convert a NodeList into an Array?
 
+    Converting an a NodeList into an array allows us to use and manipulate list in an easier way by allowing us to use native array methods like forEach(), .map() etc.
+
 5. What is a component?
+    
+    Web components are encapuslated re-usable sections code that are made up of HTML, CSS (If Styling is required) and JS(if functionality is required). 
```

Comment:
My response to this one: "A component is the intertwined html/css/js that is used to make objects on the window smarter. A component is said to be the intertwined use of html/css/js because the js code that makes the window smarter refers to both the html and css of the window to make an object do what you want it to do (e.g. adding/removing a class name to a class in js in order to add/remove a specific style in the css.)"